README.md

```diff
@@ -27,20 +27,36 @@ Title case.
 
 #### NHS, NHS.UK, NHS England
 
-Use ‘NHS.UK’ for products that are for delivery teams and not the public (find out more on the [NHS health writing A-Z](https://service-manual.nhs.uk/content/a-to-z-of-nhs-health-writing#N))This includes the NHS.UK prototype kit, NHS.UK design system, NHS.UK service manual or NHS.UK website.
+
+We are changing how we talk about tools for delivery teams. These used to be called 'NHS.UK' products but we are now removing the '.UK'. This means that we talk about the:
+
+* NHS frontend (though this is still sometimes called 'NHS.UK frontend')
+* NHS digital service manual
+* NHS design system
+* NHS prototype kit
+
+> We are changing how we talk about the Service M
```

Comment:
Service Manual?

Reply:
whoops that was a fragment that i didn't realise was still there! removed